src/reducers/cdps.js

```diff
@@ -20,7 +20,8 @@ const defaultCdpState = {
 
 export function getCdp(cdpId, { cdps, feeds }) {
   cdpId = cdpId.toString();
-  if (!cdps[cdpId]) return defaultCdpState;
+  feeds = feeds.filter(feed => feed !== undefined);
+  if (!cdps[cdpId] || feeds.length === 0) return defaultCdpState;
```

Comment:
The feeds reducer is throwing the error because `feed` is undefined (index.js 63). Filtering that out here prevents the error from being thrown.